collectors/nvd/collectors.py

```diff
@@ -327,6 +327,7 @@ def update_cvss_via_flawcvss(self, flaw: Flaw, item: dict) -> bool:
                         flaw.impact = self.calculate_highest_impact(
                             [new_cvss2_score, new_cvss3_score, new_cvss4_score]
                         )
+                        flaw.save(raise_validation_error=False)
```

Comment:
Why not use `update` instead of `save`?

Reply:
I think that `update` does not change `updated_dt`, so it seems that flaw was not changed.

Reply:
But it is then followed by `save` later, so it should be probably fine.
```
                if self.update_cvss_via_flawcvss(flaw, item):
                    updated_flaws.append(cve_id)
                    # no validation exceptions not to fail here
                    flaw.save(raise_validation_error=False)
```

Reply:
The automatic `updated_dt` change should be solved separately by @MrMarble so you does not have to account for it separately. Whenever `update` is enough, let us use it instead of save.